gui/src/controls/complexLine.ts

```diff
@@ -0,0 +1,294 @@
+/// <reference path="../../../dist/preview release/babylon.d.ts"/>
+
+module BABYLON.GUI {
+
+    export class ComplexLine extends Control {
+
+        private _lineWidth: number = 1;
+        private _dash: number[];
+        private _trackers: Nullable<Tracker>[];
+
+        private _minX: Nullable<number>;
+        private _minY: Nullable<number>;
+        private _maxX: Nullable<number>;
+        private _maxY: Nullable<number>;
+
+        constructor(public name?: string) {
+            super(name);
+
+            this.isHitTestVisible = false;
+            this._horizontalAlignment = Control.HORIZONTAL_ALIGNMENT_LEFT;
+            this._verticalAlignment = Control.VERTICAL_ALIGNMENT_TOP;
+
+            this._dash = [];
+            this._trackers = [];
+        }
+
+        public get dash(): Array<number> {
+            return this._dash;
+        }
+
+        public set dash(value: Array<number>) {
+            if (this._dash === value) {
+                return;
+            }
+
+            this._dash = value;
+            this._markAsDirty();
+        }
+
+        getX(index: number): string | number  {

```

Comment:
I think this should be at tracker level and we should just have getTracker(index) here

Reply:
Are you referring to get/set x/y/mesh/control? If so, I wanted to do just that but some functionality requires the control and the host. Perhaps pass the control to tracker?

Reply:
Yes. You can add the multi-line hist as parameter to Segment constructor